### ISSUE TITLE
httpclient behave wrongly with newrelic_rpm

### ISSUE THREAD

**Issue by OlivierLeCloset** [newrelic/newrelic-ruby-agent#457]
## Description
When using httpclient with newrelic_rpm, httpclient call an url emptied from its parameters.
The original uri is changed too.

When NR create a Tracer Segment, it obfuscated the URI. But this obfuscated URI became the httpclient URI 
```
uri=URI("https://www.exemple.com?params=value")
request = HTTP::Message.new_request("GET", uri, nil, nil, nil)
puts "GOOD #{request.header.request_uri}"
wrapped_request = NewRelic::Agent::HTTPClients::HTTPClientRequest.new(request)
puts "GOOD #{wrapped_request.uri}"

uri = wrapped_request.uri # OK 
segment = NewRelic::Agent::Transaction::ExternalRequestSegment.new HTTPClient, uri, "GET"
puts "BAD URL #{uri}"
```
## Expected Behavior
With or Without NewRelic, Htttpclient must call the same uri. NR doesn't have to clean the URI

## Steps to Reproduce
Example [here](https://github.com/OlivierLeCloset/min_fix_uri)

Gemfile
```
source 'https://rubygems.org'
gem "httpclient"
gem "newrelic_rpm"
```

```
#!/usr/bin/env ruby
require "bundler/setup"
require "httpclient"
require "newrelic_rpm"

url = "https://www.coliposte.fr/tracking-chargeur-cxf/TrackingServiceWS?wsdl"

uri = URI(url)
r=HTTPClient.get uri
puts "#{r.status_code} != 200"
puts "uri #{uri} is not the original uri #{url}"
```

## Your Environment
ruby 2.6.5

**Comment by OlivierLeCloset** [comment]
https://github.com/newrelic/newrelic-ruby-agent/pull/453

**Comment by tannalynn** [comment]
Hello @OlivierLeCloset 
I see you linked to the PR fix for this, and wanted to let you know that we have just merged it and have released a new version 6.13.1 that fixes this issue.
Thank you!
